src/rpcblockchain.cpp

```diff
@@ -939,7 +941,10 @@ UniValue rainbymagnitude(const UniValue& params, bool fHelp)
             CBitcoinAddress address(sGRCAddress);
 
             if (!address.IsValid())
+            {
+                if (sGRCAddress == "NA") continue;
                 throw JSONRPCError(RPC_INVALID_ADDRESS_OR_KEY, std::string("Invalid Gridcoin address: ") + sGRCAddress);
+            }
```

Comment:
Thinking about this some more, there's no guarantee that this function will succeed until Elizabeth. Can we...
```c++
if (!address.IsValid()) continue;
```
...and bail only if `setAddress.empty()`?

Reply:
I actually think that is fine. It really isn't a problem if we simply skip over an invalid address.